Source/JavaScriptCore/builtins/ModuleLoader.js

```diff
@@ -378,15 +378,17 @@ function provideFetch(key, value)
 }
 
 @visibility=PrivateRecursive
-async function loadModule(moduleName, parameters, fetcher)
+async function loadModule(key, parameters, fetcher)
 {
     "use strict";
 
     // Loader.resolve hook point.
     // resolve: moduleName => Promise(moduleKey)
     // Take the name and resolve it to the unique identifier for the resource location.
     // For example, take the "jquery" and return the URL for the resource.
```

Comment:
This comment belongs in `loadAndEvaluateModule` and `requestImportModule` now.  OTOH, is this comment still valid even in those 2 functions?